datahub-project/datahub
# Ingestion source creation cypress test

Test steps:
1. go to the ingestion page, create a snowflake source
2. verify yaml recipe is generated correctly
3. finish creating source
4. verify ingestion source details are saved correctly
5. remove ingestion source

## Checklist
